documentation/manual/src/main/asciidoc/modules/configuration.asciidoc

```diff
@@ -523,6 +504,77 @@ Add a `WEB-INF/jboss-deployment-structure.xml` in your archive with the followin
 More information about the descriptor can be found in the
 https://docs.jboss.org/author/display/WFLY10/Class+Loading+in+WildFly[WildFly documentation].
 
+===== Server provisioning via other build tools
+
+The provisioning tools have been developed mainly for Maven users, but there are options for users of other build tools too.
+
+Gradle users can use the link:https://plugins.gradle.org/plugin/org.wildfly.build.provision[org.wildfly.build.provision] plugin, available on the official Gradle plugin portal.
+
+Both the Maven and the Gradle plugins are just thin wrappers invoking a standalone provisioning library, so you could also integrate this library with any other JVM based tool.
```

Comment:
I'm not sure I understand what you are suggesting here. I would add it at the end of the paragraph if needed